history/characters/10000_orc.txt

```diff
@@ -336,6 +336,7 @@
 		trait = being_undead
```

Comment:
First death knights don't change race traits.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/149533464-a0c3310f-9cb2-446e-871e-06e419ef388f.png)

</details>

Reply:
Changed this for Teren'gor and Ragnok
![image](https://user-images.githubusercontent.com/35965798/149587543-7eedad43-2983-41f9-b3c1-20bf204a6b56.png)